docs/tutorials.md

```diff
@@ -212,3 +212,118 @@ curl -k --ipv4 https://02.pilosa.local:10502/index/sample-index/query -d 'Bitmap
 #### What's Next?
 
 Check out our [Administration Guide](https://www.pilosa.com/docs/latest/administration/) to learn more about making the most of your Pilosa cluster and [Configuration Documentation](https://www.pilosa.com/docs/latest/configuration/) to see the available options to configure Pilosa.
+
+
+### Integer Fields Values
+
+#### Introduction
+
+Pilosa can store integer values associated to the columns in an index, and those values are used to support range and aggregate queries. In this tutorial we will show how to set up integer fields, populate those fields with data, and query the fields. The example index we're going to create will represent fictional patients at a medical facility and various bits of information about those potients.
+
+First, create an index called `patients`:
+```
+curl localhost:10101/index/patients \
+     -X POST 
+```
+
+Next, create a frame in the `patients` index called `measurements` which will represent information gathered about each patient.
+```
+curl localhost:10101/index/patients/frame/measurements \
+     -X POST \
+     -d '{"options":{"rangeEnabled": true}}'
+```
+
+In addition to storing rows of bits, a frame can also contain fields that store integer values. The next step creates three fields (`age`, `weight`, `tcells`) in the `measurements` frame.
+```
+curl localhost:10101/index/patients/frame/measurements/field/age \
+     -X POST \
+     -d '{"type": "int", "min": 0, "max": 120}'
+
+curl localhost:10101/index/patients/frame/measurements/field/weight \
+     -X POST \
+     -d '{"type": "int", "min": 0, "max": 500}'
+
+curl localhost:10101/index/patients/frame/measurements/field/tcells \
+     -X POST \
+     -d '{"type": "int", "min": 0, "max": 2000}'
+```
+
+<div class="note">
```

Comment:
You should be able to use a shortcode here instead of html syntax, but I don't know if there's a practical difference.

Reply:
I was just looking at the note shortcode. Looks like a completely different style. The `<div class="note">` styling was provided by us for use in docs by the designers.

I think I do like the idea of using shortcodes in general though, and I think we should create a `note` one for use in docs. One advantage of the shortcode is that you can have markdown within the shortcode blocks, unlike with HTML where everything inside tags is treated literally (not parsed with Markdown).

Just for the record, the note shortcode I'm referring to is created by wrapping the content with `{{< note >}} ...
 {{< /note >}}`

Reply:
And actually I'm not so sure this block needs to be in a "note" div, as it's not a self-contained blurb, but a descriptor for the next block of text (code). Notes are generally more parenthetical.